Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+.PHONY: install superset venv pre-commit
+
+install: venv superset pre-commit
+
+superset:
+	# Install external dependencies
+	pip install -r requirements/local.txt
+
+	# Install Superset in editable (development) mode
+	pip install -e .
+
+	# Create an admin user in your metadata database
+	superset fab create-admin
+
+	# Initialize the database
+	superset db upgrade
+
+	# Create default roles and permissions
+	superset init
+
+	# Load some data to play with
+	superset load-examples
+
+venv:
+	# Create a virtual environment and activate it (recommended)
+	python3 -m venv venv # setup a python3 virtualenv
```

Comment:
Later we should add a `check` target that runs before all other targets, verifying that the user has the proper Python version installed.

Reply:
Have we considered naming the superset virtual environment something other than `venv` (perhaps specific to this application)? This reduces the chance it will conflict with existing virtual environments on folks' machines.